modules/apiserversource-yaml.adoc

```diff
@@ -140,22 +137,22 @@ spec:
       kind: Service
       name: event-display
 ----
-+
-After you have created the `trigger.yaml` file, apply it:
+
+. Apply the `Trigger` YAML file:
 +
 [source,terminal]
 ----
-$ oc apply -f trigger.yaml
+$ oc apply -f <filename>
 ----
 
-. To create events, launch a Pod in the default namespace:
+. Create events by launching a pod in the default namespace. You can do this by entering the following command:
```

Comment:
Create events by launching a pod in the default namespace by entering the following command:

Reply:
Ended up just removing the enter the command bit til we have steady guidelines on it 🙂